Helper/Price.php

```diff
@@ -192,20 +192,49 @@ private function getConfigurableProductPrice($product, $type)
      */
     private function getMinimumComplexProductPrice($product, $usedProds, $type)
     {
-        $prices = [];
+        $minimum_price = null;
+        $minimum_variant = null;
+
+        /*
+        We identify the variant with the minimum final price (or price), and
+        that is the one that is used to obtain the requested price for the
+        parent as the variant is chosen as the representative for the product
+        */
         foreach ($usedProds as $child) {
             if ($child->getId() != $product->getId()) {
-                $price = $child->getPriceInfo()->getPrice($type);
-                $prices['prices'][] =  $price;
-                $prices['values'][] =  $price->getAmount()->getValue();
+                $variant_minimum_price = $this->getMinimumVariantPrice($child);
+
+                if ($minimum_price === null) {
```

Comment:
Although this is valid, another alternatives are:

`!isset($minimum_price)` or `is_null($minimum_price)`

Just FYI

Reply:
Gracias por la sugerencia David! Me gusta más el is_null también así que lo añado!